### PR TITLE
Moises.botarro/adding fields process agent cmdline scrubbing

### PR DESCRIPTION
**Pull request by moisesbotarro**
Added the necessary fields to be set on the datadog.conf and datadog.yaml files to enable/disable the process args scrubbing feature on the agent 5 and 6, respectively.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +82,10 @@ Example Playbooks
       log_level: INFO
       apm_enabled: "true" # has to be set as a string
       logs_enabled: true  # log collection is available on agent 6
+      process_config: # on agent 6
```
Comment (conorbranagan) [inline README.md]: I think all of this part is for agent 6?

File: README.md [hunk 2]
```diff
@@ -70,6 +82,10 @@ Example Playbooks
...
+        enabled: "true" # to enable process-agent
```
Comment (conorbranagan) [inline README.md]: Maybe say `has to be set as a string. possible values are: "true", "false" (for only container collection), "disabled" to disable the process-agent entirely`

File: README.md [hunk 3]
```diff
@@ -70,6 +82,10 @@ Example Playbooks
...
+        scrub_args: true # to enable/disable process args scrubbing; default value is true
```
Comment (conorbranagan) [inline README.md]: Could be a bit clearer here, something like `enables scrubbing of sensitive arguments from the command line`

**Review by conorbranagan — COMMENTED** [review]
Added some comments on copy, lgtm otherwise

File: README.md (hunk 1 above)
Comment (moisesbotarro) [reply README.md]: The conf file also uses `datadog_config` to populate the template and the yaml template doesn't use the `datadog_config_ex` that's is presented on the Playbook Example. That's why I've put this comment here.

File: README.md [hunk 4]
```diff
@@ -52,6 +52,18 @@ Variables:
 - `datadog_agent5` - install agent5 instead of agent6 (default to `false`)
 - `datadog_agent5_apt_repo` - Override default Datadog `apt` repository for agent5
 
+To enable the process agent on agent 5, you need to set on `datadog_config`:
```
Comment (hush-hush) [inline README.md]: Should we create a specific section in the README for the process agent ? It could make thing simpler/clearer.
What do you think ?

Comment (moisesbotarro) [reply README.md]: I agree with you. It'll make the README file clearer. I've just pushed a new commit with the suggested modification

**Review by hush-hush — APPROVED** [review]
Thanks @moisesbotarro

**Pull request merged**